receiver/dockerstatsreceiver/receiver_test.go

```diff
@@ -273,6 +294,9 @@ func TestScrapeV2(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
+		if tc.desc != "scrapeV2_single_container_with_optional_resource_attributes" {
```

Comment:
Why would we not want to run the others?

Reply:
It's seems a leftover from a local debug session. Thanks for noticing! Updated: [e38b80d](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21803/commits/e38b80df5ba6cd8e2782020cadca05793e5e65fb)